docs/blender_docs/scene_gltf2.rst

```diff
@@ -318,8 +366,19 @@ a typical node structure when several of the above options are applied at once:
    A Principled BSDF material with an emissive texture.
 
 
+Exporting a Shadeless (Unlit) Material
+--------------------------------------
+
+To export an unlit material, mix in a camera ray, and avoid using the Principled BSDF node.
```

Comment:
Nearly everything in this diagram is optional, and I think it's worth mentioning that something as simple as an Image Texture or Color node connected directly to the Material Output will work — the camera ray is entirely optional.

Does this graph behave like an emissive material without the camera ray mixin?

Reply:
In Cycles it does, yes.  Also, of all the optional behaviors mentioned in the PR, this one in particular is what the importer will construct.  So I think this should round-trip more cleanly.